src/ParamConverter/UpdateConverter.php

```diff
@@ -166,7 +151,7 @@ private function validateForm(FormInterface $form, $data, Request $request)
     /**
      * @return mixed|null Returns a matching entity or 0 if nothing has been found
```

Comment:
```suggestion
     * @return mixed Returns a matching entity or null if nothing has been found
```

Reply:
Fixed in https://github.com/byWulf/apitk-manipulation-bundle/pull/12/commits/37555c8c71cc69fafa19cbfded1942048e6079e6